packages/gatsby-source-contentful/package.json

```diff
@@ -18,7 +18,7 @@
     "gatsby-plugin-sharp": "^2.1.3",
     "gatsby-source-filesystem": "^2.0.38",
     "is-online": "^7.0.0",
-    "joi": "^14.0.0",
+    "joi": "^14.3.1",
```

Comment:
It was necessary to upgrade `joi` for the sake of being able to use Joi's `oxor`